lib/configure.js

```diff
@@ -70,6 +70,9 @@ module.exports = function ToConfigure(app) {
     if (app.config.sockets.subClient) {
       app.config.sockets.adapterOptions.subClient = app.config.sockets.subClient;
     }
+    if (app.config.sockets.subEvent) {
```

Comment:
@albi34 would you take a minute to add that to the advanced options table [here on the 1.0 branch of sails-docs](https://github.com/balderdashy/sails-docs/edit/1.0/reference/sails.config/sails.config.sockets.md)?

needs type, the default value, and a description, e.g. 

> Optional.  The name of the Redis client event that Socket.io will subscribe to.
(based on https://github.com/socketio/socket.io-redis#adapteropts)

Thanks!